stl/inc/limits

```diff
@@ -1013,20 +1015,83 @@ public:
 };
 
 #ifdef __cpp_lib_bitops // TRANSITION, VSO-1020212
+

```

Comment:
The comments in `<bit>` apply to functions here; I'll mention only unique issues.